doc/concepts/data-concepts/provenance.md

```diff
@@ -0,0 +1,67 @@
+# Provenance
+
+Provenance enables Pachyderm users to go back in time and see the state of
+data at a particular moment in the past and possibly use it in root cause
+analysis, as well as to improve their code. Provenance tracks the
+dependency between datasets and determines their origins. Therefore,
+provenance answers not only the question of where the data comes from,
+but also how the data was transformed along the way. Data scientists need
+to have confidence in the information with which they operate. They need
+to be able to reproduce the results and sometimes go through the whole
+data transformation process from scratch multiple times, which makes data
+provenance one of the most critical aspects of data analysis. If your
+computations result in unexpected numbers, the first place to look for clues
+is the historical data that gives insights into possible flaws in the
+transformation chain or the data itself.
+
+Consider the following example. When a bank makes a decision about a mortgage
+application, many factors are taken into consideration, including the credit
+credit history, annual income, and so on. This data goes through multiple
+automated steps of analysis with numerous dependencies and decisions made
+along the way. If the final decision does not satisfy the applicant,
+the historical data is the first place to look for proof of authenticity,
+as well for possible prejudice against the applicant. Data provenance
+enables data scientists to track the
+progress from its origin to the final decision and make appropriate
+changes that address issues. With the adoption of General Data
+Protection Regulation (GDPR) compliance requirements, monitoring data lineage
+is becoming a necessity for many financial,
+biopharmaceutical, and other organizations that work with sensitive data.
```

Comment:
A bit awkward. Maybe: ...is becoming a necessity for many organizations that work with sensitive data (including financial and biopharmaceutical)